### PR TITLE
chore(docstrings): fixing deprecated `root` argument description in LeRobotDataset class

### PR DESCRIPTION
**Pull request by CarolinePascal**
## Type / Scope

- **Type**: Chore
- **Scope**: LeRobotDataset

## Summary / Motivation

Updates the description of the `root` parameter in `LeRobotDataset` class. Since `LeRobotDataset` v.2.0, `root` directly points to the folder in which the dataset is stored. **No `repo_id` will be added to the provided path.**

## Related issues

- Fixes / Closes: \
- Related: https://github.com/huggingface/lerobot/issues/534, https://github.com/huggingface/lerobot/pull/3030, https://github.com/huggingface/lerobot/pull/2739, https://github.com/huggingface/lerobot/pull/1057, https://github.com/huggingface/lerobot/issues/2316

## What changed

- `LeRobotDataset` docstrings.

## How was this tested (or how to run locally)

N/A

## Checklist (required before merge)

- [ ] Linting/formatting run (`pre-commit run -a`)
- [ ] All tests pass locally (`pytest`)
- [ ] Documentation updated
- [ ] CI is green

## Reviewer notes

- Anything the reviewer should focus on (performance, edge-cases, specific files) or general notes.
- Anyone in the community is free to review the PR.

### CODE REVIEW

File: src/lerobot/scripts/lerobot_dataset_viz.py [hunk 1]
```diff
@@ -132,10 +132,15 @@ def visualize_dataset(
 
     logging.info("Logging to Rerun")
 
+    first_index = None
     for batch in tqdm.tqdm(dataloader, total=len(dataloader)):
+        if first_index is None:
+            first_index = batch["index"][0].item()
         # iterate over the batch
+        print(batch["index"])
+        print(batch["index"] - first_index)
```
Comment (imstevenpmwork) [inline src/lerobot/scripts/lerobot_dataset_viz.py]: Avoid `print` in src code

Comment (CarolinePascal) [reply src/lerobot/scripts/lerobot_dataset_viz.py]: Fixed in [a293eb2](https://github.com/huggingface/lerobot/pull/3035/commits/a293eb2f97d8811bc7361b74772db58d5535b34b)
